source/_docs/terminus/plugins.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Terminus Manage Plugins
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+permalink: docs/terminus/:basename/
+---
+Extend the functionality of Terminus and add commands by installing third party plugins.
+## Install Plugins
+Add plugins within the `$HOME/.terminus/plugins` directory on your local workstation. You may need to create the `$HOME/.terminus/plugins` directory if it does not already exist.
+<div class="panel panel-default">
+  <div class="panel-heading">
+  <a data-proofer-ignore data-toggle="collapse" data-target="#advance-installs"><h3 class="panel-title" style="cursor:pointer;">Explore Advanced Install Methods (Optional) <span class="caret"></h3></a>
+  </div>
+<div id="advance-installs" class="collapse" style="padding:10px;">
+<h3>Install via Composer</h3>
+<p>Plugins published on Packagist are available to install via the Composer package manager. From a terminal window on your computer, use the following commands:</p>
+<div class="copy-snippet">
+<button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-plugin-install-composer"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+<figure><pre id="terminus-plugin-install-composer"><code class="bash" data-lang="bash">$HOME/.terminus/plugins/terminus-plugin-example
+composer create-project pantheon-systems/terminus-plugin-example
```

Comment:
Change this to:
```
cd $HOME/.terminus/plugins/terminus-plugin-example
composer create-project pantheon-systems/terminus-plugin-example:~1
```
(Missing `cd` on first line, and version designation, `:~1` on second line.)